apps/ai-content-generator/src/hooks/config/useSaveConfigHandler.ts

```diff
@@ -26,11 +28,30 @@ const useSaveConfigHandler = (
 
     const currentState = await sdk.app.getCurrentState();
 
+    let currentEditorInterface = {};
+
+    if (currentState !== null) {
+      const { EditorInterface } = currentState;
+      currentEditorInterface = EditorInterface;
+    }
+
+    // Assign the app to the sidebar for saved content types
+    const newEditorInterfaceAssignments = generateEditorInterfaceAssignments(
+      currentEditorInterface,
+      contentTypes,
```

Comment:
here is where you would use `Array.from(contentTypes)` to get access to that array